tools/gfx/vulkan/vk-command-encoder.h

```diff
@@ -54,6 +54,16 @@ class ResourceCommandEncoder
     , public PipelineCommandEncoder
 {
 public:
+    virtual void* getInterface(SlangUUID const&) { return this; }
```

Comment:
Shouldn't this be checking the uuid? It's going to return as a valid interface whatever guid is passed in(!)

Reply:
These exists mainly for compatibility of CPU Slang and isn't really intended to be called by the user, so I took the simplified path here.

Reply:
Ok. I think would be better to do something more sensible, but can approve.